[ROUTING] Integration env won't work

For now integration tests are constantly failing with the `Error while reading from Writer: bufio.Scanner: token too long` error. Error persists for all apps on `mainnet-milestone1` branches. To reproduce, simple run the env and start any test. For example:

1. `make integration-build`
2. `make integration-run-generic`
3. `./integration/startup.sh`
4. run test for any app

There's also panic on one of the nodes:

```
[2019-08-27T11:33:57+03:00] INFO [snet.dmsgC]: (HANDSHAKE) Accepted by remote. remote=02603d53d49b6575a0b8cee05b70dd23c86e42cd6cba99af769d61a6196ea2bcb1:36
[2019-08-27T11:33:57+03:00] INFO [snet.dmsgC]: Injected [ACK] received=<type:ACK><id:56958><size:2><seq:14174> remoteClient=02603d53d49b6575a0b8cee05b70dd23c86e42cd6cba99af769d61a6196ea2bcb1:36
[2019-08-27T11:33:57+03:00] INFO [snet.dmsgC]: (HANDSHAKE) Accepted locally. remote=02603d53d49b6575a0b8cee05b70dd23c86e42cd6cba99af769d61a6196ea2bcb1:~
[2019-08-27T11:33:57+03:00] INFO [snet.dmsgC]: Accepted [REQUEST] received=<type:REQUEST><id:56031><size:184> remoteClient=02603d53d49b6575a0b8cee05b70dd23c86e42cd6cba99af769d61a6196ea2bcb1 remoteServer=035915c609f71d0c7df27df85e
c698ceca0cb262590a54f732e3bbd0cc68d89282
[2019-08-27T11:33:57+03:00] INFO [snet.dmsgC]: Injected [FWD] bufSize="7/65535" received=<type:FWD><id:56031><size:9><seq:36741> remoteClient=02603d53d49b6575a0b8cee05b70dd23c86e42cd6cba99af769d61a6196ea2bcb1:~
[2019-08-27T11:33:57+03:00] INFO [route_manager]: handling setup request: setupPK(02603d53d49b6575a0b8cee05b70dd23c86e42cd6cba99af769d61a6196ea2bcb1)
[2019-08-27T11:33:57+03:00] INFO [route_manager]: Got new Setup request with type RequestRouteID
[2019-08-27T11:33:57+03:00] INFO [snet.dmsgC]: Injected [ACK] received=<type:ACK><id:56031><size:2><seq:51378> remoteClient=02603d53d49b6575a0b8cee05b70dd23c86e42cd6cba99af769d61a6196ea2bcb1:~
[2019-08-27T11:33:57+03:00] INFO [route_manager]: successfully handled setup request: setupPK(02603d53d49b6575a0b8cee05b70dd23c86e42cd6cba99af769d61a6196ea2bcb1)
panic: runtime error: index out of range

goroutine 71 [running]:
encoding/binary.bigEndian.Uint64(...)
        /usr/local/Cellar/go/1.12.9/libexec/src/encoding/binary/binary.go:124
github.com/skycoin/skywire/pkg/routing.Rule.Expiry(0x2ffc0ef, 0x0, 0x0, 0x2ffc0eb, 0xc0004437a8, 0xc0004437b0)
        /Users/nkryuchkov/GolandProjects/skywire/pkg/routing/rule.go:46 +0x1e3
github.com/skycoin/skywire/pkg/router.(*managedRoutingTable).Cleanup.func1(0x9, 0x2ffc0ef, 0x0, 0x0, 0xc000443701)
        /Users/nkryuchkov/GolandProjects/skywire/pkg/router/managed_routing_table.go:37 +0x90
github.com/skycoin/skywire/pkg/routing.(*boltDBRoutingTable).RangeRules.func1.1(0x2ffc0eb, 0x4, 0x4, 0x2ffc0ef, 0x0, 0x0, 0x0, 0x0)
        /Users/nkryuchkov/GolandProjects/skywire/pkg/routing/boltdb_routing_table.go:88 +0x118
go.etcd.io/bbolt.(*Bucket).ForEach(0xc000099b40, 0xc00042c6e0, 0x7, 0x7)
        /Users/nkryuchkov/go/pkg/mod/go.etcd.io/bbolt@v1.3.3/bucket.go:388 +0x1cc
github.com/skycoin/skywire/pkg/routing.(*boltDBRoutingTable).RangeRules.func1(0xc0003c81c0, 0x1935978, 0xc0003c81c0)
        /Users/nkryuchkov/GolandProjects/skywire/pkg/routing/boltdb_routing_table.go:94 +0x10a
go.etcd.io/bbolt.(*DB).View(0xc000108a00, 0xc00004ee50, 0x0, 0x0)
        /Users/nkryuchkov/go/pkg/mod/go.etcd.io/bbolt@v1.3.3/db.go:725 +0xb7
github.com/skycoin/skywire/pkg/routing.(*boltDBRoutingTable).RangeRules(0xc00009e208, 0xc000443760, 0x0, 0x1fa2c28)
        /Users/nkryuchkov/GolandProjects/skywire/pkg/routing/boltdb_routing_table.go:85 +0x78
github.com/skycoin/skywire/pkg/router.(*managedRoutingTable).Cleanup(0xc0001aab40, 0xc00004ef40, 0x2)
        /Users/nkryuchkov/GolandProjects/skywire/pkg/router/managed_routing_table.go:36 +0x18a
github.com/skycoin/skywire/pkg/router.(*routeManager).rtGarbageCollectLoop(0xc0000a0840)
        /Users/nkryuchkov/GolandProjects/skywire/pkg/router/route_manager.go:146 +0x1d9
created by github.com/skycoin/skywire/pkg/router.(*routeManager).Serve
        /Users/nkryuchkov/GolandProjects/skywire/pkg/router/route_manager.go:70 +0x51
```

--- a/pkg/app/app.go
+++ b/pkg/app/app.go
@@ -219,7 +219,6 @@ func (app *App) serveConn(loop routing.Loop, conn io.ReadWriteCloser) {
 		if err != nil {
 			break
 		}
-		fmt.Println("READ:", buf)
 
 		packet := &Packet{Loop: loop, Payload: buf[:n]}
 		if err := app.proto.Send(FrameSend, packet, nil); err != nil {

--- a/pkg/router/route_manager.go
+++ b/pkg/router/route_manager.go
@@ -8,15 +8,13 @@ import (
 	"net"
 	"time"
 
-	"github.com/skycoin/skywire/pkg/snet"
-
+	"github.com/google/uuid"
 	"github.com/skycoin/dmsg/cipher"
-
-	"github.com/skycoin/skywire/pkg/setup"
-
 	"github.com/skycoin/skycoin/src/util/logging"
 
 	"github.com/skycoin/skywire/pkg/routing"
+	"github.com/skycoin/skywire/pkg/setup"
+	"github.com/skycoin/skywire/pkg/snet"
 )
 
 type RMConfig struct {
@@ -308,7 +306,8 @@ func (rm *routeManager) loopClosed(data []byte) error {
 }
 
 func (rm *routeManager) occupyRouteID() ([]routing.RouteID, error) {
-	routeID, err := rm.rt.AddRule(nil)
+	rule := routing.ForwardRule(time.Now().Add(RouteTTL), 0, uuid.UUID{}, 0)
+	routeID, err := rm.rt.AddRule(rule)
 	if err != nil {
 		return nil, err
 	}

--- a/pkg/routing/rule.go
+++ b/pkg/routing/rule.go
@@ -216,5 +216,5 @@ func ForwardRule(expireAt time.Time, nextRoute RouteID, nextTrID uuid.UUID, requ
 	rule = append(rule, nextTrID[:]...)
 	rule = append(rule, bytes.Repeat([]byte{0}, 25)...)
 	binary.BigEndian.PutUint32(rule[50:], uint32(requestRouteID))
-	return Rule(rule)
+	return rule
 }

--- a/pkg/routing/rule.go
+++ b/pkg/routing/rule.go
@@ -1,6 +1,7 @@
 package routing
 
 import (
+	"bytes"
 	"encoding/binary"
 	"errors"
 	"fmt"
@@ -194,11 +195,11 @@ func AppRule(expireAt time.Time, respRoute RouteID, remotePK cipher.PubKey, remo
 	rule[8] = byte(RuleApp)
 	binary.BigEndian.PutUint32(rule[9:], uint32(respRoute))
 	rule = append(rule, remotePK[:]...)
-	rule = append(rule, 0, 0, 0, 0, 0, 0, 0, 0)
+	rule = append(rule, bytes.Repeat([]byte{0}, 8)...)
 	binary.BigEndian.PutUint16(rule[46:], uint16(remotePort))
 	binary.BigEndian.PutUint16(rule[48:], uint16(localPort))
 	binary.BigEndian.PutUint32(rule[50:], uint32(requestRouteID))
-	return Rule(rule)
+	return rule
 }
 
 // ForwardRule constructs a new forward RoutingRule.
@@ -213,7 +214,7 @@ func ForwardRule(expireAt time.Time, nextRoute RouteID, nextTrID uuid.UUID, requ
 	rule[8] = byte(RuleForward)
 	binary.BigEndian.PutUint32(rule[9:], uint32(nextRoute))
 	rule = append(rule, nextTrID[:]...)
-	rule = append(rule, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0)
+	rule = append(rule, bytes.Repeat([]byte{0}, 25)...)
 	binary.BigEndian.PutUint32(rule[50:], uint32(requestRouteID))
 	return Rule(rule)
 }

